examples/ffd/ffd_cyberon.cmake

```diff
@@ -0,0 +1,194 @@
+set(FFD_SRC_ROOT ${CMAKE_CURRENT_LIST_DIR})
+
+set(MODEL_LANGUAGE "english_usa")
+set(CYBERON_COMMAND_NET_FILE "${FFD_SRC_ROOT}/model/english_usa/Hello_XMOS_pack_WithTxt.bin.Enc.NibbleSwap")
+
+#**********************
+# Gather Sources
+#**********************
+file(GLOB_RECURSE APP_SOURCES ${CMAKE_CURRENT_LIST_DIR}/src/*.c )
+
+
+set(APP_INCLUDES
+    ${CMAKE_CURRENT_LIST_DIR}/src
+    ${CMAKE_CURRENT_LIST_DIR}/src/gpio_ctrl
+    ${CMAKE_CURRENT_LIST_DIR}/src/intent_engine
+    ${CMAKE_CURRENT_LIST_DIR}/src/intent_handler
+    ${CMAKE_CURRENT_LIST_DIR}/src/intent_handler/audio_response
+    ${CMAKE_CURRENT_LIST_DIR}/src/power
+)
+set(RTOS_CONF_INCLUDES
+    ${CMAKE_CURRENT_LIST_DIR}/src/rtos_conf
+)
+
+#**********************
+# QSPI Flash Layout
+#**********************
+set(BOOT_PARTITION_SIZE 0x100000)
+set(FILESYSTEM_SIZE_KB 1024)
+math(EXPR FILESYSTEM_SIZE_BYTES
+     "1024 * ${FILESYSTEM_SIZE_KB}"
+     OUTPUT_FORMAT HEXADECIMAL
+)
+
+set(CALIBRATION_PATTERN_START_ADDRESS ${BOOT_PARTITION_SIZE})
+
+math(EXPR FILESYSTEM_START_ADDRESS
+    "${CALIBRATION_PATTERN_START_ADDRESS} + ${LIB_QSPI_FAST_READ_DEFAULT_CAL_SIZE_BYTES}"
+    OUTPUT_FORMAT HEXADECIMAL
+)
+
+math(EXPR MODEL_START_ADDRESS
+    "${FILESYSTEM_START_ADDRESS} + ${FILESYSTEM_SIZE_BYTES}"
+    OUTPUT_FORMAT HEXADECIMAL
+)
+
+set(CALIBRATION_PATTERN_DATA_PARTITION_OFFSET 0)
+
+math(EXPR FILESYSTEM_DATA_PARTITION_OFFSET
```

Comment:
I am not sure I love having all of this magic in a cmake file but it is in line with other apps here. It's good for the first time user I guess.

No change request - just thoughts.